src/elm/Page/Device.elm

```diff
@@ -553,17 +564,40 @@ update session msg model =
                     Just ConfirmModal.Danger
 
                 modal =
-                    ConfirmCredentialsWipe (ConfirmModal.init title body action style True) UpdateConfirmModal
+                    ConfirmCredentialsWipe (ConfirmModal.init title body action style True True) UpdateConfirmModal
             in
             ( { model | currentModal = Just modal }
             , Cmd.none
             , ExternalMsg.Noop
             )
 
         WipeDeviceCredentialsDone (Ok _) ->
-            ( model
+            let
+                title =
+                    "Success"
+
+                body =
+                    Html.p
+                        []
+                        [ Html.text "The device's credentials secret was wiped from Astarte. You can "
+                        , Html.a
+                            [ href <| Route.toString <| Route.Realm <| Route.RegisterDevice (Just model.deviceId) ]
+                            [ Html.text "click here" ]
+                        , Html.text " to register the device again and store its new credentials secret."
```

Comment:
maybe instead of "store" we might go for "use".

Reply:
I went with "retrieve" instead of "store", as @rbino suggested. Would that be ok?